packages/liveblocks-core/src/__tests__/room.test.ts

```diff
@@ -1541,9 +1541,40 @@ describe("room", () => {
     });
 
     test("event", async () => {
-      const { room, wss } = createTestableRoom({});
+      const { room, wss } = createTestableRoom(
+        {},
+        undefined,
+        SOCKET_AUTOCONNECT_BUT_NO_ROOM_STATE
+      );
+      room.connect();
 
       wss.onConnection((conn) => {
+        conn.server.send(
+          serverMessage({
+            type: ServerMsgCode.ROOM_STATE,
+            actor: 2,
+            nonce: "nonce-for-actor-2",
```

Comment:
> nonce

This word has a very different meaning in the UK 😂

Reply:
So I learned… after shipping this 🙈 To be clear, its intended meaning is [from cryptography](https://en.wikipedia.org/wiki/Cryptographic_nonce).